src/tests/stub_StoreMeta.cc

```diff
@@ -11,12 +11,18 @@
 #define STUB_API "StoreMeta.cc"
 #include "tests/STUB.h"
 
+#include "sbuf/SBuf.h"
```

Comment:
Inclusion of SBuf.h is no longer necessary because no current relevant API returns SBuf. More specifically, GetNewSwapMetaVaryHeaders() API that triggered this change request became a private/static helper function in 84a0f8d.